src/test/java/gui/TestEditRulePopup.java

```diff
@@ -0,0 +1,17 @@
+package test.java.gui;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import main.java.gui.EditRulePopup;
+
+class TestEditRulePopup {
+	@Test
+	void test() {

```

Comment:
name this appropriately please. @ftfni-iscteiul may be able to help you with this

Reply:
Do we have any defined naming standart? I used the same name format as the main GUI's current testing class, that being test{ClassName}.

Reply:
I did not notice the test name, I was refering to the class name above. I've now changed the method name to be more descriptive of what the test is doing.